br/br-log-command-line.md

```diff
@@ -0,0 +1,358 @@
+---
+title: 日志备份和恢复功能使用介绍
+summary: 了解如何使用 br log 命令行工具进行日志备份。
+---
+
+# 日志备份功能使用
+
+> **警告：**
+>
+> 当前该功能为实验特性，不建议在生产环境中使用。打开该功能需要配置参数 tikv:  backup-stream.enable: true 开启该功能。
+
+## 前置条件 
+
+使用日志备份前，你需要安装 BR。你可以选择[使用 TiUP 在线安装](/migration-tools.md#使用-tiup-快速安装)（推荐），或[下载 TiDB 离线包](/download-ecosystem-tools.md)。
+
+## 使用日志备份
+
+进行日志备份时，你需要使用 `br log` 命令：
+
+```shell
+tiup br log --help
+
+backup stream log from TiDB/TiKV cluster
+
+Usage:
+  br log [command]
+
+Available Commands:
+  metadata   get the metadata of log backup storage 
+  pause      pause a log backup task
+  resume     resume a log backup task
+  start      start a log backup task
+  status     get status for the log backup task
+  stop       stop a log backup task
+  truncate   truncate the log data until sometime.
+```
+
+下面逐一介绍各个子命令的使用
+
+- `br log start` 启动一个日志备份任务
+- `br log status` 查询日志备份任务状态
+- `br log pause` 暂停日志备份任务
+- `br log resume` 重启处于暂停的备份任务
+- `br log stop` 停止备份任务, 并删除任务元信息
+- `br log truncate` 从备份存储中清理日志备份数据
+- `br log metadata` 查询备份存储中备份数据的元信息
+
+### 启动日志备份任务
+
+通过 `br log start` 在备份集群启动一个日志备份任务。该任务在 TiDB 集群持续地运行，及时地将 kv 变更日志保存到备份存储中。 运行 `br log start –help` 获取该子命令使用介绍
+
+```shell
+./br log start --help
+start a log backup task
+
+Usage:
+  br log start [flags]
+
+Flags:
+  -h, --help               help for start
+  --start-ts string        usually equals last full backupTS, used for backup log. Default value is current ts. support TSO or datetime, e.g. '400036290571534337' or '2018-05-11 01:42:23'.
```

Comment:
@joccau please update these command example, I know these many small changes for command line and its output